mitmproxy/script/concurrent.py

```diff
@@ -11,7 +11,7 @@ class ScriptThread(basethread.BaseThread):
     name = "ScriptThread"
 
 
-def concurrent(fn):
+def concurrent(fn):  # pragma: no coverage windows
```

Comment:
Right - there's no excuse why we couldn't test this on Windows, so let's either fix the test for it or exclude this file from `--full-cov` for now. This should be tested on Windows eventually.